packages/shared/routes/setup/Congratulations.svelte

```diff
@@ -82,8 +42,6 @@
                      * because the last app the user had open was the legacy one
                      */
                     promptUserToConnectLedger(false, () => $appRouter.next())
```

Comment:
we shouldnt route here, the function should look something like

```
    const exportMigrationLog = (): void => {
        getProfileDataPath($activeProfile.id)
            .then((source) =>
                isLedgerProfile
                    ? Platform.exportLedgerMigrationLog($migrationLog, `${$activeProfile.id}-${LOG_FILE_NAME}`)
                    : Platform.exportMigrationLog(`${source}/${LOG_FILE_NAME}`, `${$activeProfile.id}-${LOG_FILE_NAME}`)
            )
            .catch(console.error)
            .finally(() => (exportStrongholdBusy = false))
    }
```
and it could be combined with the logic below too